clang/lib/CIR/CodeGen/CIRGenCoroutine.cpp

```diff
@@ -276,7 +274,9 @@ CIRGenFunction::buildCoroutineBody(const CoroutineBodyStmt &S) {
                        /*ArraySize=*/nullptr);
 
   auto storeAddr = coroFrame.getPointer();
-  builder.create<mlir::cir::StoreOp>(openCurlyLoc, nullPtrCst, storeAddr);
+  auto coroPtrTy = storeAddr.getType().dyn_cast<mlir::cir::PointerType>();
+  auto temp = builder.createBitcast(nullPtrCst, coroPtrTy.getPointee());
```

Comment:
Why do we need this bitcast here? Can't the coroFrame be defined in terms of void ptr instead?

Reply:
I suppose it can, and I think it already is.

I probably forgot to remove this `temp` var. I'll fix it.